src/utils/calculateMilliseconds.ts

```diff
@@ -6,8 +6,8 @@
  * @param {string} unit The unit of time to convert to milliseconds.
  * @returns {number} The number of milliseconds.
  */
-export const calculateMilliseconds = (value: number, unit: string) => {
-  switch (unit) {
+export const calculateMilliseconds = (value: number, unit: string): number => {
+  switch (unit.toLowerCase()) {
```

Comment:
```suggestion
export const calculateMilliseconds = (value: number, unit: "seconds" | "minutes" | "hours" | "days" | "weeks"): number => {
  switch (unit) {
```

We could actually get more strict with the typing and avoid potential issues.

Reply:
Thank you.

Reply:
However we can't actually do that as I just found out. Because the mute command will fail verification.